src/citrine/_utils/template_util.py

```diff
@@ -0,0 +1,61 @@
+from gemd.util.impl import recursive_flatmap
+from gemd.entity.object import (
+    ProcessSpec,
+    ProcessRun,
+    MaterialSpec,
+    MeasurementSpec,
+    MeasurementRun,
+)
+from gemd.entity.attribute import PropertyAndConditions
+
+
+def make_attribute_table(gems: list):
+    """[ALPHA] the current status of make_attribute_table.
+
+    Given a list of GEMD Objects, this method returns a list of dictionaries
+    where in each dictionary there are keys of "object", "object_type" and keys for each
+    unique Attribute Type + Name pair found within those Objects.
+    Each value contains the associated Object, Object Type, or BaseValue.
+    This dictionary can easily be converted into a Pandas DataFrame where there is a
+    row for each Object, and the values within the cells are that Object's type or
+    the various GEMD Value Types.
+
+    This function will help users identify attribute usage across their Objects,
+    and get a sense of the consistency of the data found in their GEMD objects.
+    With the returned dictionary, one can easily do further analysis on both
+    attribute values and value type consistency to assist in data cleaning and
+    Attribute Template creation.
+
```

Comment:
Can we add parameters and return value for a full numpy-style docstring (like [here](https://github.com/CitrineInformatics/citrine-python/blob/main/src/citrine/builders/design_spaces.py#L297))?